fluent.ebnf

```diff
@@ -32,7 +32,7 @@ quoted-text          ::= ([^{"] | '\{' | '\"')+
 block-text           ::= NL __ '|' unquoted-pattern
 
 placeable            ::= '{' __ expression __ '}'
-expression           ::= selector-expression | select-expression
+expression           ::= selector-expression | select-expression | variants-list
```

Comment:
the reference to it is, but the definition is nowhere to be found. Did I miss something?